modulemd_add_platform/README

```diff
@@ -0,0 +1,5 @@
+This tool edits a modulemd-packager YAML document.
```

Comment:
Can you please add this file as `README.md` and add a heading to it, to be consistent with other tools?

Reply:
I will do it, but your advice damages Python package metadata. A content of README is included into long description. That will pollute it with markdown codes and a useless, duplicate title.